pipeline/tgpolyt_depths.py

```diff
@@ -17,7 +17,8 @@
 from common.misc import open_file, have_file
 
 # location of TG-polyT in CFTR reference
-REGION = ['CFTR', 87824, 87852, 'TG-polyT']
+#REGION = ['CFTR', 87824, 87852, 'TG-polyT']
+REGION = ['7', 117188670, 117188690, 'TG-polyT']
```

Comment:
Will this ever change? Would be good to have it as 3 options with these as the default values